hca2scea-backend/test_hca2scea.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+import unittest
+import pandas as pd
+
+class CharacteristicTest(unittest.TestCase):
+
+    def setUp(self):
+        self.verificationErrors = {}
+        if not sys.warnoptions:
+            import warnings
+            warnings.simplefilter("ignore")
+
+    def test_hca2scea_characteristic(self):
+        # run tool
+        arguments_df = pd.read_csv("test/golden/arguments.txt", sep="\t")
+        for i in range(0,arguments_df.shape[0]):
+            spreadsheet = "test/golden/" + list(arguments_df['spreadsheet'])[i]
+            with self.subTest(spreadsheet="test/golden/" + list(arguments_df['spreadsheet'])[i]):
+                arguments = arguments_df.loc[arguments_df['spreadsheet'] == os.path.basename(spreadsheet)]
+                output_dir = self.run_tool(spreadsheet, arguments)
+                self.check_output(output_dir, spreadsheet)
+
+    def get_content(self, golden_file, output_file):
```

Comment:
It is better for a function to do one thing because it can be tested more easily, reused more easily and understood more easily.

Therefor, I would change to something like:

```python
def get_file_content(file):
   # read contents of one file here

```

and use it twice:
```python
output_contents = get_file_contents(output_file)
golden_contents = get_file_contents(golden_file)
```